### DISCUSSION TITLE
Site not loading properly

### DISCUSSION THREAD

**Discussion by TheFlashSpeedster** [PrivateBin/PrivateBin#1364] [Q&A]
### Describe the problem/question

I was having issues in Network. It was blocking css, stylesheet, img, font. So, i put this CSP Header
![image](https://github.com/PrivateBin/PrivateBin/assets/78632620/42f75d64-e6a8-49b5-95cb-5b04404a87d4)

Now, In Inspect > Network > All Requests are Unblocked

But, site isn't working as expected. menu aren't working, text box disappeared, etc.
Also, Check my Console Errors
![image](https://github.com/PrivateBin/PrivateBin/assets/78632620/6c357c1b-445d-4770-8f3c-e278a8abd7df)

### Did you use the FAQ section?

- [x] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

1. download source code
2. upload and extract to your site
3. visit site (I didnt moduify any files)

### What happens

loading screen...
menu not working
Multiple Errors an Warnings in Console

### What should happen

Site should load and show text box and menu button should work, etc...

### Server address

https://paste.pc-games.eu.org

### PrivateBin version

v1.6.2 anf v1.7.1

### Browser and version

firefox v127.0

### Local operating system and version

Windows 10

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by elrido** [comment]
I would raise a ticket with your CDN provider, they clearly mess with your content, inject additional code and break your application. Notice that the actual CSP header sent is "default-src *;  ; script-src * 'nonce-7459031d-8b0d-47da-a8e3-fa25bde2dc10'", additional meta-tags and inline scripts get injected and this mysterious s.js gets loaded next, which is not one of ours either. Have them turn that off for this subdomain, if possible, and otherwise have them fix their own induced bugs.

It seems you are in a similar situation to the cloudflare victims: https://github.com/PrivateBin/PrivateBin/wiki/FAQ#user-content-how-to-make-privatebin-work-when-using-cloudflare-for-ddos-protection
